docker-compose.yml

```diff
@@ -30,7 +30,29 @@ services:
         protocol: tcp
         mode: host
 
+  s3:
+    image: minio/minio
+    volumes:
+      - minio_data:/export
+    ports:
+      - 9000:9000
+    command: server /export
+
+  s3_createbuckets:
+    image: minio/mc
+    depends_on:
+      - s3
+    entrypoint: >
+      /bin/sh -c "
+      /usr/bin/mc alias set myminio http://s3:9000 minio Welkom01;
+      /usr/bin/mc mb myminio/ticketvise;
+      /usr/bin/mc policy set download myminio/ticketvise;
+      exit 0;
+      "
```

Comment:
Gaat dit goed? Je overwrite niks in de prod

Reply:
Ik zie inderdaad dat de access keys niet kloppen